### PR TITLE
BRC 5 Cards

### PR DESCRIPTION
**Pull request by Simisays**
This should wrap up Brothers War Commander except those two cards that need engine changes

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Staff of Titania
+ManaCost:2
+Types:Artifact Equipment
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ X | AddToughness$ X | Description$ Equipped creature gets +X/+X, where X is the number of Forests you control. 
+T:Mode$ Attacks | ValidCard$ Creature.EquippedBy | Execute$ TrigToken | TriggerDescription$ Whenever equipped creature attacks, create a 1/1 green Forest Dryad land creature token. (It's affected by summoning sickness.)
+SVar:TrigToken:DB$ Token | TokenScript$ g_1_1_forest_dryad_land
+K:Equip:3
+SVar:HasAttackEffect:TRUE
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt:8]: This SVar should be on the creature

File: forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Staff of Titania
+ManaCost:2
+Types:Artifact Equipment
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ X | AddToughness$ X | Description$ Equipped creature gets +X/+X, where X is the number of Forests you control. 
+T:Mode$ Attacks | ValidCard$ Creature.EquippedBy | Execute$ TrigToken | TriggerDescription$ Whenever equipped creature attacks, create a 1/1 green Forest Dryad land creature token. (It's affected by summoning sickness.)
+SVar:TrigToken:DB$ Token | TokenScript$ g_1_1_forest_dryad_land
+K:Equip:3
+SVar:AE:HasAttackEffect:TRUE
+SVar:X:Count$TypeYouCtrl.Forest
+DeckHas:Ability$Token Type$Dryad
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt:10]: ` & `

File: forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Staff of Titania
+ManaCost:2
+Types:Artifact Equipment
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ X | AddToughness$ X | Description$ Equipped creature gets +X/+X, where X is the number of Forests you control.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt:4]: So this just needs `AddSVar$ AE`

File: forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Name:Staff of Titania
+ManaCost:2
+Types:Artifact Equipment
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ X | AddToughness$ X | Description$ Equipped creature gets +X/+X, where X is the number of Forests you control. 
+T:Mode$ Attacks | ValidCard$ Creature.EquippedBy | Execute$ TrigToken | TriggerDescription$ Whenever equipped creature attacks, create a 1/1 green Forest Dryad land creature token. (It's affected by summoning sickness.)
+SVar:TrigToken:DB$ Token | TokenScript$ g_1_1_forest_dryad_land
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt:6]: Looks like this token script isn't made yet

File: forge-gui/res/cardsfolder/upcoming/the_archimandrite.txt [hunk 5]
```diff
@@ -0,0 +1,15 @@
+Name:The Archimandrite
+ManaCost:2 U R W
+Types:Legendary Creature Human Advisor
+PT:0/5
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigGainLife | TriggerDescription$ At the beginning of your upkeep, you gain X life, where X is the number of cards in your hand minus 4.
+SVar:TrigGainLife:DB$ GainLife | LifeAmount$ X
+T:Mode$ LifeGained | TriggerZones$ Battlefield | ValidPlayer$ You | Execute$ TrigPumpAll| TriggerDescription$ Whenever you gain life, each Advisor, Artificer, and Monk you control gains vigilance and gets +X/+0 until end of turn, where X is the amount of life you gained.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/the_archimandrite.txt:7]: spacing

File: forge-gui/res/cardsfolder/upcoming/the_archimandrite.txt [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+SVar:TrigPumpAll:DB$ PumpAll |  ValidCards$ Advisor.YouCtrl,Artificer.YouCtrl,Monk.YouCtrl | NumAtt$ +Y | KW$ Vigilance
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/the_archimandrite.txt:8]: spacing

File: forge-gui/res/cardsfolder/upcoming/titania_natures_force.txt [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+ManaCost:4 G G
+Types:Legendary Creature Elemental
+PT:6/6
+S:Mode$ Continuous | Affected$ Forest.YouOwn | MayPlay$ True | AffectedZone$ Graveyard | Description$ You may play Forests from your graveyard.
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Forest.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever a Forest enters the battlefield under your control, create a 5/3 green Elemental creature token.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ g_5_3_elemental | TokenOwner$ You
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Elemental.YouCtrl | OptionalDecider$ You | Execute$ TrigMill | TriggerDescription$ Whenever an Elemental you control dies, you may mill three cards.
+SVar:TrigMill:DB$ Mill | NumCards$ 3 | Defined$ You
+DeckHints:Type$Elemental
+DeckHas:Ability$Graveyard|Mill
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/titania_natures_force.txt:11]: maybe a `SVar:BuffedBy:Forest` ?

File: forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt (hunk 4 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt:6]: https://github.com/Card-Forge/forge/pull/1784/files#diff-e3c2cd5d005981d2a00d57b8790d29a61baa99e7f0d50c28f93644bc2e447e4b

File: forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt [hunk 8]
```diff
@@ -1,11 +1,11 @@
 Name:Staff of Titania
 ManaCost:2
 Types:Artifact Equipment
-S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ X | AddToughness$ X | Description$ Equipped creature gets +X/+X, where X is the number of Forests you control. 
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddSVar$ AE | AddPower$ X | AddToughness$ X | Description$ Equipped creature gets +X/+X, where X is the number of Forests you control. 
 T:Mode$ Attacks | ValidCard$ Creature.EquippedBy | Execute$ TrigToken | TriggerDescription$ Whenever equipped creature attacks, create a 1/1 green Forest Dryad land creature token. (It's affected by summoning sickness.)
 SVar:TrigToken:DB$ Token | TokenScript$ g_1_1_forest_dryad_land
 K:Equip:3
 SVar:AE:HasAttackEffect:TRUE
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt:8]: `SVar:AE:SVar:HasAttackEffect:TRUE`

**Review by tool4ever — APPROVED** [review]

**Pull request merged**